geode-lucene/src/upgradeTest/java/org/apache/geode/cache/lucene/RollingUpgradeQueryReturnsCorrectResultsAfterClientAndServersAreRolledOverAllBucketsCreated.java

```diff
@@ -31,7 +29,7 @@
 public class RollingUpgradeQueryReturnsCorrectResultsAfterClientAndServersAreRolledOverAllBucketsCreated
     extends LuceneSearchWithRollingUpgradeDUnit {
 
-  @Test
+  // @Test
```

Comment:
Actually, add `@Ignore` and just leave `@Test` in place. You co do something like:
```
@Ignore("Disabled until GEODE-7258 is fixed")
```
And then add a note to GEODE-7258 to re-enable this test.

Reply:
Done.